packages/server/postgres/pgmConfig.js

```diff
@@ -0,0 +1,12 @@
+
+const pgmConfig = {
```

Comment:
@tiffanyhan the pgm docs are...not good. basically, i just debugged their code until i figured out how it was parsing this file & now it's working!